test/error-handling-test-data.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+export default {
+    invalidValuesResponse: {
+      code: 400,
+      message: "Invalid organization id provided",
+      details: []
+    },
+    unauthorizedResponse: {
+      code:401,
+      status:"Unauthorized",
+      message:"The request could not be authorized"
+    },
```

Comment:
We added support for 403 recently. You should merge beta into your unit-test branch to get that update and add unit tests for 403 as well.